src/domain/repository/runtimeCmdRepo.go

```diff
@@ -6,6 +6,7 @@ import (
 )
 
 type RuntimeCmdRepo interface {
+	RestartPhp() error
```

Comment:
Why? Why can't you use the Restart from the ServiceCmdRepo? Why would you need to restart the php if the php is only "restartable" if you restart the php-webserver?